packages/devextreme/js/__internal/ui/html_editor/modules/m_resizing.ts

```diff
@@ -206,8 +206,16 @@ export default class ResizingModule extends BaseModule {
     }
 
     if (option === 'enabled') {
+      if (this.enabled === value) {
```

Comment:
I guess it's better to just use optional chaining [here](https://github.com/DevExpress/DevExtreme/pull/27707/files#diff-5a4765a2721171612bfac242bfc75cb9ffbb5f1f3406ac090246e6f31e3a79eeR226), don't u think? Because I'm not sure this private cache `enabled` is always updated properly. Particularly, I'm not sure what happens if we pass `null/undefined`

Reply:
ok let me check

Reply:
[bba349a](https://github.com/DevExpress/DevExtreme/pull/27707/commits/bba349a51ff1a26fc7bee4b232459520d1a7e683)